docs/contracts.rst

```diff
@@ -128,7 +128,7 @@ Visibility and Getters
 Since Solidity knows two kinds of function calls (internal
 ones that do not create an actual EVM call (also called
 a "message call") and external
-ones that do), there are four types of visibilities for
```

Comment:
Please also remove the opening parenthesis above.

Reply:
Reverted. Although nested parentheses aren't great, it's the clearest option here.